PubSub/Sources/API/TopicsAPI.swift

```diff
@@ -0,0 +1,66 @@
+import Core
+import NIO
+import AsyncHTTPClient
+import Foundation
+
+public protocol TopicsAPI {
+    func get(topicId: String) -> EventLoopFuture<GoogleCloudPubSubTopic>
```

Comment:
Can we add docs/comments about the parameters for using the API? The models aren't necessarily too important at the moment but the actual API calls would be useful.

Reply:
Done.